test/forms/common/DeclarationPageHelper.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.common
+
+import forms.DeclarationPage
+import models.DeclarationType.{DeclarationType, _}
+import models.viewmodels.TariffContentKey
+import models.DeclarationType
+import org.scalatest.{MustMatchers, WordSpec}
+
+trait DeclarationPageHelper extends WordSpec with MustMatchers {
```

Comment:
Nit: I'd not extend these test traits here to avoid coupling.

Reply:
:+1: